src/util/cache.ts

```diff
@@ -0,0 +1,16 @@
+import NodeCache from 'node-cache';
+
+export class CacheUtil {
+  constructor(protected cacheService = new NodeCache()) {}
+
+  // TTL in seconds
+  public set<T>(key: string, value: T, ttl = 3600): boolean {
+    return this.cacheService.set(key, value, ttl);
+  }
+
+  public get<T>(key: string): T {
+    const value = this.cacheService.get(key);
```

Comment:
I see that the `cacheService.get` accepts a generic type, instead of doing `value as T`you could do like `cacheService.get<T>(key)`

Reply:
That's better!